drag_and_drop_v2/utils.py

```diff
@@ -22,7 +21,7 @@ def ngettext_fallback(text_singular, text_plural, number):
         return text_plural
 
 
-def _clean_data(data):
+def clean_data(data):
```

Comment:
@abodacs Can your kindly revert this and other related renames?

Reply:
@tecoholic 
Done Thanks